src/features/session_replay/aggregate/index.js

```diff
@@ -213,6 +216,8 @@ export class Aggregate extends AggregateBase {
   getHarvestContents () {
     const agentRuntime = getRuntime(this.agentIdentifier)
     const info = getInfo(this.agentIdentifier)
+    const firstTimestamp = this.timestamp.event.first || this.timestamp.cycle.first
+    const lastTimestamp = this.timestamp.event.last || this.timestamp.cycle.last
```

Comment:
Edge case where either the `event.first` or `event.last` exists but not the other? Then calculation would be using one of `event` and one of `cycle`. Is that guaranteed to be impossible?

Reply:
Not an issue